frontend/packages/ux-editor/src/components/TextResource.module.css

```
@@ -1,9 +1,12 @@
 .root {
   --frame-offset: 4px;
-  border-radius: 2px;
+  border-radius: 1rem;
+  padding: 1rem;
   display: flex;
   flex-direction: column;
-  gap: 1rem;
+  gap: 4px;
+  margin-top: 8px;
+  background-color: white;
```

Should we use variable from FDS? 😊 
```suggestion
  background-color: var(--fds-semantic-surface-action-first-no_fill)
```